### PR TITLE
various improvements from editorial pass

### PR DESCRIPTION
**Pull request by dhh1128**
Signed-off-by: dhh1128 <daniel.hardman@gmail.com>

### CODE REVIEW

File: docs/spec-files/encryption.md [hunk 1]
```diff
@@ -33,32 +35,32 @@ In addition, DIDComm defines optional implementation usage of the draft [XC20P](
...
+DIDComm Messaging's guarantees with respect to man-in-the-middle attacks are easily misunderstood, since DIDs rather than human identity are the concern of the algorithm. Please refer to the [DIDComm Guidebook](https://didcomm.org/book/v2/mitm) for details.
 
-### Key IDs `kid` and `skid` headers references in the DID document
+#### Key IDs `kid` and `skid` headers references in the DID document
 
-Keys used by DIDComm envelopes MUST be sourced from the DIDs exchanged between two agents. Specifically, both sender and recipients keys MUST be retrieved from the DID document's `KeyAgreement` verification section as per the [DID Document Keys](https://identity.foundation/didcomm-messaging/spec/#did-document-keys) definition.
+Keys used by DIDComm envelopes MUST be sourced from the DIDs exchanged between two agents. Specifically, both sender and recipient encryption keys MUST be retrieved from the respective DID document's `keyAgreement` verification section as per the [DID Document Keys](https://identity.foundation/didcomm-messaging/spec/#did-document-keys) definition.
```
Comment (TelegramSam) [inline docs/spec-files/encryption.md:63]: This change from `KeyAgreement` to `keyAgreement` is good, but there are plenty of nearby references not updated. Should we address that as part of a different PR, or do you want to change the others as well?

File: docs/spec-files/protocols.md [hunk 2]
```diff
@@ -18,56 +20,43 @@ protocol-identifier-uri = doc-uri delim protocol-name "/" semver
...
-Standardizing its format is important because it is parsed by agents that
-will map messages to handlers--basically, code will look at this string and
-say, "Do I have something that can handle this message type inside protocol
-*X* version *Y*?" When that analysis happens, it must do more than compare
-the string for exact equality; it may need to check for semver compatibility,
-and it has to compare the protocol name and message type name ignoring case
-and punctuation.
-
-The URI MUST be composed as follows:
+A __Message Type URI__ (MTURI) identifies message types unambiguously.Standardizing its format is important because it is parsed by agents that will map messages to handlers &mdash; basically, code will look at this string and say, "Do I have something that can handle this message type inside protocol *X* version *Y*?" When that analysis happens, it must do more than compare the string for exact equality; it may need to check for semver compatibility, and it has to compare the protocol name and message type name ignoring case and punctuation.
```
Comment (TelegramSam) [inline docs/spec-files/protocols.md:50]: >unambiguously.Standardizing
missing space in the edit here.

Comment (dhh1128) [reply docs/spec-files/protocols.md]: Fixed.

File: docs/spec-files/encryption.md (hunk 1 above)
Comment (dhh1128) [reply docs/spec-files/encryption.md:63]: Fixed.

**Comment by mccown** [comment]
I read through the comments and think they look good.

**Comment by mccown** [comment]
Approved in the DIDComm WG on 20220404

**Pull request merged**